hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -247,11 +247,12 @@ public static void main(final String... args) throws Exception {
                 () -> {
                     isGenesis.set(true);
                     final var genesisState = hedera.newMerkleStateRoot();
+                    final var genesisNetwork = DiskStartupNetworks.fromLegacyAddressBook(diskAddressBook);
```

Comment:
For now I left as it is in https://github.com/hashgraph/hedera-services/pull/16843, since `diskAddressBook` is used in some more places above and I didn't want to have some information coming from `config.txt` and some coming from the new `json` file format.